h2o-core/src/main/java/water/api/Schema.java

```diff
@@ -441,6 +448,8 @@ public S fillFromParms(Properties parms, boolean checkRequiredFields) {
         splits = new String[] { s.trim() };
       }
 
+      if (afclz==int.class || afclz==double.class || afclz==float.class)
```

Comment:
Was this a bug? Can there be a test to demonstrate the fix?

Reply:
Yes, it is a bug.  Without my fix, the newly added DL example will fail to run.

Reply:
Can you please investigate how does the JSON request look like? Could you please paste it here for reference?